docs/src/usage/numpy.rst

```diff
@@ -58,6 +58,7 @@ The squaring of ``x`` occurs externally to MLX, meaning that no gradient is inco
 It's important to note that a similar issue arises during array conversion and copying.
 For instance, a function defined as ``mx.array(np.array(x)**2).sum()`` would also result in an incorrect gradient,
 even though no in-place operations on MLX memory are executed.
+``mx.array`` supports dlpack as well.
```

Comment:
This is not the right spot for this. Too buried. I will make a suggestion further up.

Reply:
Can you do something like at Line 6?

```
MLX array supports conversion between other frameworks through either:

1. The `Python Buffer Protocol <https://docs.python.org/3/c-api/buffer.html>`_. 
2. `DLPack <https://dmlc.github.io/dlpack/latest/>`_.

Let's convert an array to NumPy and back.
```

Reply:
I have made the requested changes. Thank you!!